robots/gazebo_tank.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+// BoB robotics includes
+#include "../common/pose.h"
+#include "../common/stopwatch.h"
+#include "tank.h"
+
+// Third-party includes
+#include "../third_party/units.h"
+#include <utility> 
+namespace BoBRobotics {
+namespace Robots {
+using namespace units::literals;
+
+template<typename LengthUnit = units::length::millimeter_t,
+         typename AngleUnit = units::angle::degree_t>
```

Comment:
We're not using these templated types anywhere, so I'd drop the templating. (Tbh I'm not sure it was even a good idea when I used it for the ``SimulatedTank`` class...)